src/api/v1beta1/feature_flags_resources.go

```diff
@@ -0,0 +1,92 @@
+package v1beta1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	annotationFeatureActiveGate = annotationFeaturePrefix + "activegate-"
+	annotationFeatureEec        = annotationFeatureActiveGate + "eec-"
+	annotationFeatureStatsd     = annotationFeatureActiveGate + "statsd-"
+)
+
+// FeatureEecResourcesRequests is a feature flag to define CPU or memory requests for the EEC container
+func (dk *DynaKube) FeatureEecResourcesRequests(resourceName corev1.ResourceName) *resource.Quantity {
```

Comment:
I understand that we can only enable statsd and eec with an annotation so we need other annotations to configure stuff for it.

But, this kinda feels wrong in a way. 
If we want to give the customer a new fully configurable activegate feature, why don't we just put it in the `activeGate` section, which already has resource limit/requests ?

(The PR with the `extensions` annotations is different because that adds config options that would require changes to the CR otherwise, which we would like to avoid.)

Reply:
It was discussed on several occassions with @baichinger, @LukaszHalman and @chrismuellner, with contradictory conclusions. The most recent one was to avoid any changes to CRD and go with feature flags first, mostly due to the ongoing discussions about ActiveGate list (a.k.a. `v1beta2`).

To quote Bernhard (see IF-94, Feb 9th), 
> we must not introduce any new fields or changes to the CRD. Otherwise we end up in a release deadlock - a no-go from my point of view. 

Pity I can't link to our internal JIRA from here...

So:
- in the long run we want StatsD (or, in general, Extensions)-related properties to make it into CRD once we're confident about their stability
- for now we just want to release the feature without messing up the CRD too much (as format migration are already a headache).